src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Button({buttonText, stateButtonResetFunction,buttonType}) {
+    return (
+        <button
+            onClick={stateButtonResetFunction}
+            type={buttonType}
+        >
+            {buttonText}
+        </button>
+    );
+}
+
+export default Button;
```

Comment:
Dit component is nu een doorgeefluik. Zelf zou ik hier niet een apart component voor maken, maar gewoon direct het html button element gebruiken. Het zorgt nu niet perse voor code reductie namelijk